### PR TITLE
docs: added issue templates as form🛠

### LINKED ISSUE #42: Add Issues Templates

### PR DESCRIPTION
**Pull request by MrKrishnaAgarwal**
Signed-off-by: Krishna Agarwal <dmkrishna.agarwal@gmail.com>

## Describe your changes
Same as Title
## Issue ticket number and link - If Any
closes #42 
## Checklist before requesting a review

- [x] I have performed a self-review of my code.

- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).

- [x] I ran the app and tested it locally to verify that it works as expected.

- [x] I have starred the repository.

### CODE REVIEW

**Comment by MrKrishnaAgarwal** [comment]
_cc @devarshishimpi_

**Review by devarshishimpi — APPROVED** [review]

**Comment by devarshishimpi** [comment]
@MrKrishnaAgarwal Could you summarize the required labels again??

**Comment by MrKrishnaAgarwal** [comment]
> @MrKrishnaAgarwal Could you summarize the required labels again??

Yes, just a sec

**Comment by MrKrishnaAgarwal** [comment]
> @MrKrishnaAgarwal Could you summarize the required labels again??

just add the `other` label
